App.tsx

```diff
@@ -120,38 +123,50 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                             units={'/100'}
                             IconClass={A}
                             fontSize={20}
-                            iconColor={PXBColors.green[500]}
+                            iconColor={theme.dark ? PXBColors.green[200] : PXBColors.green[500]}
+                        />
+                        <Hero
+                            label={'Battery'}
+                            value={'Full'}
+                            IconClass={Battery}
+                            iconColor={theme.dark ? PXBColors.blue[200] : PXBColors.blue[500]}
                         />
-                        <Hero label={'Battery'} value={'Full'} IconClass={Battery} iconColor={PXBColors.blue[500]} />
                         <Hero label={'Estimated'} IconClass={Clock} iconColor={PXBColors.gray[500]}>
                             <ChannelValue fontSize={20} value={1} units={'h'} />
                             <ChannelValue fontSize={20} value={37} units={'m'} />
                         </Hero>
-                        <Hero label={'Loaded'} IconClass={Pie} iconColor={PXBColors.blue[500]}>
+                        <Hero
+                            label={'Loaded'}
+                            IconClass={Pie}
+                            iconColor={theme.dark ? PXBColors.blue[200] : PXBColors.blue[500]}
+                        >
                             <ChannelValue fontSize={20} value={65} units={'%'} IconClass={ChartLineVariant} />
                         </Hero>
                         <Hero
                             label={'Not Shown'}
                             value={'5th Item'}
                             IconClass={Battery}
-                            iconColor={PXBColors.blue[500]}
+                            iconColor={theme.dark ? PXBColors.blue[200] : PXBColors.blue[500]}
                         />
                     </HeroBanner>
                     <InfoListItem
                         divider={'full'}
                         IconClass={Sunny}
+                        iconColor={theme.colors.text}
```

Comment:
What's the default color being used by the infoListItem for the iconColor? Is it not the theme.colors.text already?

Reply:
Here's the implementation for the infoListItem iconColor
` const getIconColor = useCallback((): string => {
        if (iconColor) return iconColor;
        if (avatar) {
            return statusColor
                ? color(statusColor).isDark()
                    ? Colors.white[50]
                    : Colors.black[500]
                : Colors.white[50]; // default avatar is dark gray -> white text
        }
        return statusColor ? statusColor : theme.colors.text;
    }, [iconColor, avatar, statusColor]);`

Looks like if there is no supplied iconColor, avatar, or statusColor then it should use theme.colors.text but I'm getting Colors.black[500] regardless of the theming (this is the theme.colors.text for the light theme). I'm wondering if the theme isn't getting updated for that function since it's not included in the dependencies.

Reply:
> I'm wondering if the theme isn't getting updated for that function since it's not included in the dependencies.

👍